components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -165,6 +167,7 @@ abstract class AbstractFetchDownloadService : Service() {
                         setDownloadJobStatus(currentDownloadJobState, PAUSED)
                         currentDownloadJobState.job?.cancel()
                         emitNotificationPauseFact()
+                        logger.info("ACTION_PAUSE for ${currentDownloadJobState.state.url} PAUSED")
```

Comment:
Let's use `debug` instead of `info`? Seems like the perfect case for it :)

Reply:
Should `logger.debug` logs only be available on debug builds and `logger.info` on all builds?

Reply:
Would mind help me to clarify, I'm bit confused in that aspect as I'm trying to log statements that could help to access an issue even if the app is a release build.

Reply:
Cases like this one https://github.com/mozilla-mobile/fenix/issues/12297, where we don't have enough information to determine what is the issue.

Reply:
@Amejia481 Pretty much the same. Hooking up `adb logcat` you will see these as debug log statements which is exactly what you want. You're using them to debug a problem. You don't need a debug build to see them. You can try running `adb logcat` and opening Fenix beta/release. You will see statements marked with "D".

Reply:
Thanks 👍